layouts/community/ergodox/meagerfindings/config.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CONFIG_MEAGERFINDINGS_H
+#define CONFIG_MEAGERFINDINGS_H
+
+#include QMK_KEYBOARD_CONFIG_H
```

Comment:
Could you remove this? 

It doesn't actually do anything.

Reply:
Removed